sys-libs/libxcrypt/libxcrypt-4.4.19.ebuild

```diff
@@ -52,19 +66,50 @@ src_configure() {
 	multibuild_foreach_variant multilib-minimal_src_configure
 }
 
+get_xcsysroot() {
+	if is_cross; then
+		echo "${EPREFIX}/usr/${CTARGET}"
```

Comment:
ESYSROOT is probably more appropriate here, right? And then we can just drop this helper altogether?

Reply:
(This [post](https://mgorny.pl/articles/the-ultimate-guide-to-eapi-7.html#improved-cross-compilation-support) is really helpful)

Reply:
crossdev will create an ebuild named cross-/libxcrypt e.g. cross-armv7a-linux-gnueabihf/libxcrypt. The expectation is building this package will install files to /usr/armv7a-linux-gnueabihf/. Use of CTARGET is used to install to correct location i.e. /usr/${CTARGET}.

So ESYSROOT is not right.

Reply:
I just verified the documentation as well as how the cross- code works in practice and what @m-gupta noted above is correct.

Reply:
I'm still not actually convinced ${ESYSROOT}/usr/${CTARGET} is wrong. Just unusual.

The final destination for my system isn't necessarily where deps are. ESYSROOT is not for where our final location is.

Reply:
All that said, binutils seems to either do it wrong too, so it's not a huge deal. I'll sort them all out together at some point, I guess.